### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by EatWooloos**
- Added battlepedia search to /dome/neopets.phtml
- Fixed auctions to include genie searches
- Formatting fixes for quest page

### CODE REVIEW

File: searchhelper.user.js [hunk 1]
```diff
@@ -462,6 +478,17 @@ if (isBeta) {
...
+    
+    // Battledome
+    if (document.URL.includes("/dome/neopets")) {
+        $(".equipFrame").each(function (index, element) {
+            const itemname = $(element).text().trim();
+            if (itemname) { // ignore if empty slot
+                $(makelinks(itemname)).appendTo($(element));
+            }
+        });
+        $(".equipTable").css({"overflow" : "scroll"});
```
Comment (diceroll123) [inline searchhelper.user.js:490]: `overflow-y`, since it adds one for x which isn't needed

File: searchhelper.user.js [hunk 2]
```diff
@@ -27,6 +28,7 @@
 // @match      http://www.neopets.com/island/tradingpost.phtml*
 // @match      http://www.neopets.com/generalstore.phtml*
 // @match      http://www.neopets.com/faerieland/hiddentower938.phtml
+// @match      http://www.neopets.com/dome/neopets.phtml
```
Comment (diceroll123) [inline searchhelper.user.js:31]: Add an asterisk here, `phtml*`

**Comment by diceroll123** [comment]
Once those two requests are fulfilled, we're ready for merging! 😄 thanks!

**Comment by EatWooloos** [comment]
Done! Perhaps could consider using the battlepedia icons for all pages, not just limited to battledome page? I left this out in the release to reduce clutter on non-battledome items, but have since changed to personally use it for all pages and prefer it more.

**Comment by diceroll123** [comment]
Mmm, perhaps 🤔

In time I'd like to make a little settings menu to give the user better control of what icons show. People have privately asked me how to get rid of the trading/auctions icons, etc.

**Pull request merged**